partcad/src/partcad/wrappers/wrapper_openscad.py

```
@@ -0,0 +1,80 @@
+import os
+import shutil
+import tempfile
+import sys
+
+sys.path.append(os.path.dirname(__file__))
+import wrapper_common
+
+
+def process(path, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `process`

(ANN201)